octobot_commons/tentacles_management/advanced_manager.py

```diff
@@ -190,8 +190,13 @@ def get_all_classes(clazz, config) -> list:
 
 
 def get_all_classes_from_parent(parent_class) -> list:
-    return [subclass if not subclass.__subclasses__() else get_all_classes_from_parent(subclass)
-            for subclass in parent_class.__subclasses__()]
+    classes = []
```

Comment:
?

Reply:
It was returning lists of lists in some cases instead of a list with all children

Reply:
Ok :+1: